index.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-nav');
```

Comment:
[OPTIONAL] I observed numerous commented codes in your project that have not been implemented in this PR. It is recommended to remove these superfluous sections if they are not necessary.